### ISSUE TITLE
Add Ukrainian translation

### ISSUE THREAD

**Issue by msberends** [msberends/AMR#64]
Had contact with @gxpcluster to add Ukrainian translation. Fantastic news! We'll gladly incorporate it into the package.

There's 2 ways to go forward:
1. Just paste the contents of an edited version of [`data-raw/translations.tsv`](https://github.com/msberends/AMR/blob/main/data-raw/translations.tsv) here, containing an extra column "uk" (we follow [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1)) with the translated strings. We can then merge it into our project manually.
2. Fork this project (I see this is already done, great!), edit the file [`github.com/gxpcluster/AMR/data-raw/translations.tsv`](https://github.com/gxpcluster/AMR/blob/main/data-raw/translations.tsv) and create a Pull Request. We can then merge it into our project automatically.

For us, the second option would be ideal, but the first is less work for you probably, which is also perfectly fine for us.

If there are any questions, please let me know!

**Comment by gxpcluster** [comment]
Hi Matthijs.

Thank you for the instructions. I will try the second option and send a pull request. Is it better to use development or master branch?

We also have a question about the phrase "Antimicrobial Interpretation". If I understand correctly, this is the label for the x-axis on the plot. Could you please explain this phrase, because we have several options for its translation and we would like to choose the best one.

We have also found that plots have another labels and titles that are not presented in **tsv** file. Such as:
- _Percentage_
- _MIC values of_
- _Resistance Overview of_
- _Disk zones of_

([here](https://msberends.github.io/AMR/reference/plot.html))
Can we add these phrases to the edited file?
I would also like to ask you to attach one or two examples of such plots.

Thank you.

**Comment by msberends** [comment]
Good points! I’ll try to respond to your questions in the next coming week.

**Comment by gxpcluster** [comment]
Hi, Matisse!
We completed the translation of the tsv file, but would like to wait for your response to make the final corrections.
We look forward to hearing from you.

**Comment by gxpcluster** [comment]
Dear Matthijs,
We completed the translation of the tsv file, but would like to wait for your response to make the final corrections.
We look forward to hearing from you.

**Comment by msberends** [comment]
That's great news!

Please use the development branch to update `translations.tsv`. We will update the current git flow to not rely on the development branch, it's not very convenient and the website is not generated automatically at the moment. This will be solved using GitHub Actions in the next months.

Regarding your questions:

> We also have a question about the phrase "Antimicrobial Interpretation". If I understand correctly, this is the label for the x-axis on the plot. Could you please explain this phrase, because we have several options for its translation and we would like to choose the best one.

Indeed, it is used for the x-axis when plotting R/SI values using either `plot()`, `barplot()` (both base R), or `autoplot()` (ggplot2). This is the only occurrence, so it will be used like this to give you an example:

``` r
library(AMR)
values <- random_rsi(100)
barplot(values)
```

![image](https://user-images.githubusercontent.com/31037261/185111935-16c2454d-d6d8-42b1-adbc-24ed4ce16900.png)

```r
ggplot2::autoplot(values)
```

![image](https://user-images.githubusercontent.com/31037261/185112015-7a3bf62f-5c8b-483d-ae6a-a619ac89f5ed.png)


> We have also found that plots have another labels and titles that are not presented in tsv file. Such as:
> 
> * Percentage
> * MIC values of
> * Resistance Overview of
> * Disk zones of

Nice catch! Only "Percentage" must be translated, the other 3 are leftovers and will be removed in a future version. I will add the translation of "Percentage" of the other languages.

The only place where "Percentage" is being used is when plotting R/SI values using `plot()`:

```r
plot(values)
```

![image](https://user-images.githubusercontent.com/31037261/185112626-26277f4a-3444-41d7-8af4-e25d43caffc4.png)

**Comment by gxpcluster** [comment]
Dear Matthijs,

Thank you for all the explanations. We will introduce corresponding changes, and today I will try to make a pull request.

**Comment by gxpcluster** [comment]
Pull request is done. Everything seems to be ok.

**Comment by msberends** [comment]
Fixed in #67
